src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -319,3 +320,64 @@ g.test('stencil_fail_operation')
     ];
     t.runStencilStateTest(baseState, testStates, 2, _expectedColor);
   });
+
+g.test('stencil_disabled')
+  .desc(
+    `
+  Test that for all combinations, the last triangle drawn is the one visible if the stencil is
+  disabled. We need to check against three triangles because the stencil test may modify results.
```

Comment:
This description doesn't make much sense out of context. It needs to include an overview before talking about small details.

That said this test doesn't really cover anything about stencil testing. Without enabling some stencil state, there's no way to ever update the stencil buffer, so we wouldn't really be able to tell if there was a bug where stencil _testing_ state was wrong.

The original test is the same way, it wasn't testing anything about stencil, but it _was_ meaningfully testing something about depth: it draws triangles at 3 different depths in 3 different orders and ensures that depth testing is not taking place.

So I think if we want to keep this test it needs to be moved to the depth state tests. The stencil state test file has enough coverage, I think, that I'm not concerned about the default "disabled" states being wrong.

Reply:
> The original test is the same way, it wasn't testing anything about stencil, but it was meaningfully testing something about depth: it draws triangles at 3 different depths in 3 different orders and ensures that depth testing is not taking place.

> So I think if we want to keep this test it needs to be moved to the depth state tests. The stencil state test file has enough coverage, I think, that I'm not concerned about the default "disabled" states being wrong.


It's likely I was not enough to understand the original test. I'm going to move this test to depth.spec.ts with some updates.